pkg/ipam/crd.go

```diff
@@ -340,6 +384,11 @@ func (n *nodeStore) updateLocalNodeResource(node *ciliumv2.CiliumNode) {
 	defer n.mutex.Unlock()
 
 	if n.conf.IPAMMode() == ipamOption.IPAMENI {
+		if err := validateENIConfig(node); err != nil {
+			log.WithError(err).Warning("ENI state is not consistent yet")
```

Comment:
I wonder if we should lower this to `Info`. In particular if it can happen in CI it will cause flakes in the future (Paul is working on re-enabling fail-on-warning for CI).